standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -10165,14 +10165,46 @@ public NotificationEventsCountResponse getNotificationEventsCount(NotificationEv
       long fromEventId = rqst.getFromEventId();
       String inputDbName = rqst.getDbName();
```

Comment:
Can use db and catalog name in lower case or need to check if it works with any case.

Reply:
Similar to the above comment, by following the code, I confirmed that the names are always lower case. So, like above would it work if we assert that they lower case to save some cycles?

Reply:
ok, assert is fine.

Reply:
Per our offline discussion fixed getNotificationEventsCount() so that it converts the database name into lower case if it's not * pattern, in which case the condition on database name is excluded from the query.